test/test_image.py

```diff
@@ -190,6 +191,8 @@ def test_decode_png_errors():
         decode_png(torch.empty((), dtype=torch.uint8))
     with pytest.raises(RuntimeError, match="Content is not png"):
         decode_png(torch.randint(3, 5, (300,), dtype=torch.uint8))
+    with pytest.raises(RuntimeError, match="Internal error"):
```

Comment:
Hmm it seems that calling the `png_error()` method leads to a RunTime error with not helpful error message.

Reply:
```suggestion
    with pytest.raises(RuntimeError, match="Out of bound read in decode_png"):
```